### PR TITLE
Added helper text when the console is empty. Fixes https://github.com/mozilla/thimble.mozilla.org/issues/2091

### PR DESCRIPTION
**Pull request by flukeout**
When there are no log entries in the console, now we show a handy message so people can figure out how to use it.

Should appear like this when console is empty...

image

Fixes https://github.com/mozilla/thimble.mozilla.org/issues/2091

### CODE REVIEW

File: locales/en-US/editor.properties [hunk 1]
```diff
@@ -214,6 +214,8 @@ ERROR_MOVING_FILE_SAME_NAME=A file or folder with the name {0} already exists in
 CONSOLE_TITLE=Console
 # Tooltip that shows when hovering over the console icon in the Thimble preview
 CONSOLE_TOOLTIP=Open the JavaScript Console
+# Text that shows up when the console is empty with instructions on how to use it
+CONSOLE_HELPTEXT=To use the console, add  <code>console.log("Hello World!");</code> to your JavasScript file.
```
Comment (humphd) [inline locales/en-US/editor.properties]: Nit: 2 spaces after `add`, should be 1

File: src/extensions/default/bramble/htmlContent/console.html [hunk 2]
```diff
@@ -4,5 +4,9 @@
         <button id="clearConsole" title="{{CONSOLE_CLEAR_TOOLTIP}}" class="clear-button">{{CONSOLE_CLEAR}}</button>
         <button title="{{CONSOLE_CLOSE_TOOLTIP}}"href="#" class="close"></button>
     </div>
-    <div class="console"></div>
+    <div class="console">
+      <div class="helper">
+        <span class="helper-text">{{&CONSOLE_HELPTEXT}}</span>
```
Comment (humphd) [inline src/extensions/default/bramble/htmlContent/console.html]: What is this `&` doing here?

Comment (humphd) [inline src/extensions/default/bramble/htmlContent/console.html]: From the Mustache docs, this is actually OK to do, and I see it elsewhere in the Brackets code.  You can also do `{{{...}}}` to unescape as well.  So disregard my comment.

**Comment by flukeout** [comment]
Fixed nit & merging.

**Pull request merged**